exercises/yacht/instructions.md

```diff
@@ -1,8 +1,12 @@
-# Description
+# Instructions
 
-The dice game [Yacht][yacht] is from the same family as Poker Dice, Generala and particularly Yahtzee, of which it is a precursor.
-In the game, five dice are rolled and the result can be entered in any of twelve categories.
-The score of a throw of the dice depends on category chosen.
+Given five dice and a category, calculate the score of the dice for that category.
+
+~~~~exercism/note
+You'll always be presented with five dice.
+Each dice's value will be between one and six inclusively.
+The dice will be unordered.
```

Comment:
Not sure what the nicest way is to list these "guarantees"

Reply:
I think this is reasonable.

```suggestion
~~~~exercism/note
You'll always be presented with five dice.
Each dice's value will be between one and six inclusively.
The dice may be unordered.
```